configure.ac

```diff
@@ -192,34 +192,59 @@ AC_SUBST([QEMU_PATH], [$ac_cv_path_QEMU])
 AC_SUBST([CMDLINE], [$(cat $srcdir/data/kernel-cmdline)])
 
 #bundle path for functional-test
-DEFAULT_BUNDLE_TEST_DIR=$(cd "$srcdir/tests/functional/bundle" 2>/dev/null; pwd -P)
 AC_ARG_WITH([tests-bundle-path],
     [AS_HELP_STRING([--with-tests-bundle-path=[[BUNDLE-PATH]]],
             [path to oci bundle to execute functional tests])
     ],
     [
         BUNDLE_TEST_PATH=$(cd "$with_tests_bundle_path"; pwd -P)
         AC_SUBST([BUNDLE_TEST_PATH])
-    ],
-    [
-        AC_SUBST([BUNDLE_TEST_PATH], [$DEFAULT_BUNDLE_TEST_DIR])
     ]
 )
 
-DEFAULT_ROOTFS_PATH=$BUNDLE_TEST_PATH/rootfs
 AC_ARG_WITH([tests-rootfs-path],
```

Comment:
could we simplify the configuration here? (just an innocent question, shouldn't be part of this PR). is there an interesting case where we want ROOTFS_PATH != BUNDLE_PATH/rootfs?